share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -154,7 +154,18 @@ endfunction()
 
 function(rocm_set_soversion LIBRARY_TARGET SOVERSION)
     if(NOT WIN32)
-        rocm_version_regex_parse("^([0-9]+).*" SOVERSION_MAJOR "${SOVERSION}")
-        set_target_properties(${LIBRARY_TARGET} PROPERTIES VERSION ${SOVERSION} SOVERSION ${SOVERSION_MAJOR})
+        rocm_version_regex_parse("^([0-9]+).*" LIB_VERSION_MAJOR "${SOVERSION}")
+        rocm_version_regex_parse("^[0-9]+\\.([0-9]+).*" LIB_VERSION_MINOR "${SOVERSION}")
+
+        if (DEFINED ENV{ROCM_LIBPATCH_VERSION})
+            set (LIB_VERSION_PATCH $ENV{ROCM_LIBPATCH_VERSION})
```

Comment:
This should call the `rocm_get_build_info` function to append the build information.

Reply:
Ok. I will update the changes and call the rocm_get_build_info to the ROCM_LIBPATCH_VERSION from env.

Reply:
I have updated the way to access the ROCM_LIBPATCH_VERSION though the rocm_get_so_patch function. Please check the latest commit.